src/Kunstmaan/FormBundle/Controller/FormSubmissionsController.php

```diff
@@ -89,11 +89,16 @@ public function listAction(Request $request, $nodeTranslationId)
      */
     public function editAction($nodeTranslationId, $submissionId)
     {
-        $em              = $this->getDoctrine()->getManager();
-        $nodeTranslation = $em->getRepository('KunstmaanNodeBundle:NodeTranslation')->find($nodeTranslationId);
-        $formSubmission  = $em->getRepository('KunstmaanFormBundle:FormSubmission')->find($submissionId);
+        $em                   = $this->getDoctrine()->getManager();
+        $nodeTranslation      = $em->getRepository('KunstmaanNodeBundle:NodeTranslation')->find($nodeTranslationId);
+        $formSubmission       = $em->getRepository('KunstmaanFormBundle:FormSubmission')->find($submissionId);
+        $formSubmissionFields = $em->getRepository('KunstmaanFormBundle:FormSubmissionField')->getFieldsOrderedBySequence($formSubmission);
```

Comment:
Aha! That's indeed what I needed but didn't find right away. Will fix tomorrow. Thx!